big_transfer.py

```diff
@@ -56,13 +57,15 @@ def __init__(self,
                  model_path: Optional[str] = 'pretrained',
                  model_name: Optional[str] = 'R50x1',
                  on_gpu: bool = False,
+                 target_dim: Tuple[int] = (256, 256, 3),
```

Comment:
It might be nice, to also make this optional. If it is `None`, then the preprocessing will not happen.

Reply:
Sure, np